# Message for defining the state of the car
Header header           # header for when the message is published
bool estop_on           # whether the estop is turned on
bool mux_automatic      # whether the mux is set to autonomous or human controlled
float32 speed_mps       # speed of the car in m/s